runtime/frequency/src/lib.rs

```diff
@@ -386,6 +386,7 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 	type Event = Event;
 	type MotionDuration = CouncilMotionDuration;
 	type MaxProposals = CouncilMaxProposals;
+	 // Only one member? I guess it will be just sudo for now? To be consistent, probably move the value to constants.rs
 	type MaxMembers = ConstU32<1>;
```

Comment:
That is wrong, but it is just for weight calculations so it works that we have 5 members currently. @wilwade to add an issue for it to update it.

Reply:
https://github.com/LibertyDSNP/frequency/issues/757